### PR TITLE
Add new cluster profile aro-redhat-tenant

### PR DESCRIPTION
**Pull request by slintes**
new cluster profile for the ARO (managed Openshift on Azure) team

also see https://github.com/openshift/release/pull/56644

### CODE REVIEW

File: pkg/api/types.go [hunk 1]
```diff
@@ -1355,6 +1355,7 @@ const (
 	ClusterProfileAWSStackrox           ClusterProfile = "aws-stackrox"
 	ClusterProfileAWSSDCICD             ClusterProfile = "aws-sd-cicd"
 	ClusterProfileGCPSDCICD             ClusterProfile = "gcp-sd-cicd"
+	ClusterProfileAroRH                                = "aro-redhat-tenant"
```
Comment (smg247) [inline pkg/api/types.go:1358]:
```suggestion
	ClusterProfileAroRH                   ClusterProfile = "aro-redhat-tenant"
```

Comment (smg247) [reply pkg/api/types.go:1358]: You'll have to format that properly as well

Comment (slintes) [reply pkg/api/types.go]: oh, thx, done!

**Pull request merged**